packages/module/src/components/contextmenu/ContextSubMenuItem.tsx

```diff
@@ -77,11 +81,12 @@ const ContextSubMenuItem: React.FunctionComponent<ContextSubMenuItemProps> = ({
               e.stopPropagation();
             }
           }}
+          ref={subMenuRef}
+          role="presentation"
+          isNavFlyout
         >
-          <Dropdown toggle={() => <></>} className={css(topologyStyles.topologyContextMenuCDropdownMenu)}>
```

Comment:
Is this class still needed?

Reply:
It's also used here:

https://github.com/patternfly/react-topology/blob/dc28e8bf77e363eda3507a6b547ecafaa916a611/packages/module/src/components/contextmenu/ContextMenu.tsx#L33